File: src/main/java/com/qa/cdstore/service/CDService.java
package com.qa.cdstore.service;

public interface CDService {

	public String getAllCds();
	
	public String getSingleCD(int id);
	
	public String deleteCD(int id);
	
	public String addCD(String objJSON);
	
	public String updateCD(int id, String objSJON);
}

File: src/main/webapp/app/services/service/cdservice.js
/**
 * Created by Administrator on 17/02/2017.
 */

(function() {

    var CDService =  function(cdDal) {

        this.getCDs = function()
        {
            return cdDal.getCDs();
        };

        this.saveCD = function(cdToAdd)
        {
            return cdDal.saveCD(cdToAdd);
        };

        this.deleteCD = function(cdToRemove)
        {
            return cdDal.deleteCD(cdToRemove);
        };
    };
    
    angular.module('cdStore').service('cdService', ['cdDal', CDService]);
}());